docs-kits/kits/Business Partner Kit/Documentation BPDM/bpdm_arc42.md

```diff
@@ -592,6 +486,198 @@ UpdateLayoutConfig($c4ShapeInRow="3", $c4BoundaryInRow="1")
 
 ## Crosscutting Concepts
 
+### Authentication & Autorization
+
+### Roles, Rights, Permissions
```

Comment:
Please make relevant sub headings here, currently `Roles, Rights, Permissions` should appear in same precision of heading `Authentication & Autorization`